marklogic-data-hub/src/main/java/com/marklogic/hub/hubcentral/migration/HubCentralMigrator.java

```diff
@@ -0,0 +1,126 @@
+package com.marklogic.hub.hubcentral.migration;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.ext.helper.LoggingObject;
+import com.marklogic.hub.EntityManager;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.HubProject;
+import com.marklogic.hub.entity.DefinitionType;
+import com.marklogic.hub.entity.HubEntity;
+import com.marklogic.hub.impl.EntityManagerImpl;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class HubCentralMigrator extends LoggingObject {
+    private HubConfig hubConfig;
+    private EntityManager entityManager;
+    private FlowMigrator flowMigrator;
+    private ObjectMapper mapper = new ObjectMapper();
+
+    public HubCentralMigrator(HubConfig hubConfig) {
+        this.hubConfig = hubConfig;
+        this.entityManager = new EntityManagerImpl(hubConfig);
+        this.flowMigrator = new FlowMigrator(this.hubConfig);
+    }
+
+    /**
+     * Migrate the entity model, flow and mapping files in a user's local project. Does not make any changes to what's stored in MarkLogic.
+     */
+    public void migrateUserArtifacts() {
+        migrateFlows();
+        migrateEntityModels();
+    }
+
+    /**
+     * Migrate the entity model files in a user's local project. Does not make any changes to what's stored in MarkLogic.
+     */
+    protected void migrateFlows() {
+        flowMigrator.migrateFlows();
```

Comment:
Just call this directly in migrateUserArtifacts, the extra method adds a level of indirection without any benefit.